elife/certificates.sls

```diff
@@ -50,6 +50,10 @@ web-complete-cert:
     cmd.run:
         - name: cat certificate.crt fullchain.pem > certificate.chained.crt && chgrp {{ pillar.elife.certificates.username }} certificate.chained.crt
         - cwd: /etc/certificates/
+        # only trigger state if either of these two files have changed
```

Comment:
```
If a state should only execute when another state has changes, and otherwise do nothing, the new onchanges requisite should be used instead of watch. watch is designed to add additional behavior when there are changes, but otherwise the state executes normally.
```
This seems to fit what we need here: only re-execute the `cat` in this case.

Reply:
yes, I read and re-read that passage quite a bit ;)